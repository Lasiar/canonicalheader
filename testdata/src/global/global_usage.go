package global

func dontImportPackage() {
	header.Get("Test-HEader") // want `use "Test-Header" instead of "Test-HEader"`
}
