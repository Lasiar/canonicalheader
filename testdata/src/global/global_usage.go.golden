package global

func dontImportPackage() {
	header.Get("Test-Header") // want `use "Test-Header" instead of "Test-HEader"`
}
